src/debug/logs.py

```diff
@@ -139,20 +139,24 @@ def get_regex_list():
     regex.append([debug.on_ignored_message,
                   r"PRESPsReaderQueue_newData:!get entries"])
 
-    regex.append([debug.on_ignored_message, r"^send failed: $"])
-    regex.append([debug.on_ignored_message, r"^\s*locator:\s*$"])
-    regex.append([debug.on_ignored_message, r"^\s*transport: \d+$"])
-    regex.append([debug.on_ignored_message, r"^\s*address: [\d:]+$"])
-    regex.append([debug.on_ignored_message, r"^[\d:]+$"])
-    regex.append([debug.on_ignored_message, r"^\s*port: \d+$"])
-    regex.append([debug.on_ignored_message, r"^\s{6}encapsulation:$"])
-    regex.append([debug.on_ignored_message, r"^\s{3}transport_priority: \d+$"])
-    regex.append([debug.on_ignored_message, r'^\s{3}aliasList: ""$'])
+    regex.append([debug.on_ignored_message, r"^send failed:"])
```

Comment:
Review these changes and add the missing symbol in the regex instead of removing the `$` symbol.